content/en/docs/IV/modules/hosp/_index.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: "Hosp"
-linkTitle: "hosp"
+linkTitle: "Hosp"
 date: 2020-08-10
 weight: 20
 description: >
-  The Hosp module provides all data acquired from the hospital wide electronic health record. Information covered includes laboratory measurements, microbiology, medication administration, and billed diagnoses.
+  The Hosp module provides all data acquired from the hospital wide electronic health record. Information covered includes patients and admissions information, laboratory measurements, microbiology, medication administration, and billed diagnoses.
 ---
 
 The hosp module contains data derived from the hospital wide EHR. These measurements are predominantly recorded during the hospital stay, though some tables include data from outside the hospital as well (e.g. outpatient laboratory tests in labevents).
-Information includes laboratory measurements (labevents, d_labitems), microbiology cultures (microbiologyevents), provider orders (poe, poe_detail), medication administration (emar, emar_detail), medication prescription (prescriptions, pharmacy), hospital billing information (diagnoses_icd, d_icd_diagnoses, procedures_icd, d_icd_procedures, hcpcsevents, d_hcpcs, drgcodes), and service related information (services).
+Information includes patients and admissions informations (patients,admissions,transfers), laboratory measurements (labevents, d_labitems), microbiology cultures (microbiologyevents), provider orders (poe, poe_detail), medication administration (emar, emar_detail), medication prescription (prescriptions, pharmacy), hospital billing information (diagnoses_icd, d_icd_diagnoses, procedures_icd, d_icd_procedures, hcpcsevents, d_hcpcs, drgcodes), and service related information (services).
```

Comment:
Suggestion: "Information includes patient and admission information (patients, admissions, transfers),..."

Reply:
Fixed